pkgs/applications/networking/browsers/brave/generic.nix

```diff
@@ -0,0 +1,218 @@
+{ lib, stdenv, fetchurl, wrapGAppsHook, makeWrapper
```

Comment:
Nit: rename this file to `make-brave.nix` instead of `generic.nix`.

(Seriously, `generic.nix` is such an ugly name!)

Reply:
Lol  in my defense this implementation is based on the `signal-desktop` pkg.